### PR TITLE
Run cygpath when restarting build.py in Windows.

### PR DESCRIPTION
**Pull request by dekimir**

### CODE REVIEW

**Comment by dekimir** [comment]
@AWoloszyn @antiagainst @dneto0 @Qining PTAL

File: utils/build.py [hunk 1]
```diff
@@ -91,6 +91,10 @@ def build(args):
     run(['ctest', '--output-on-failure'], args.builddir, env, args.dry_run)
 
 
+def cygpath(f):
+    p = subprocess.Popen(['cygpath', '-w', f], stdout=subprocess.PIPE)
```
Comment (AWoloszyn) [inline utils/build.py]: Can we use check_output here instead the output will be more clear. (I can't remember if this was failing on windows, if so, ignore my comment.)

**Comment by AWoloszyn** [comment]
Other than my one comment, looks good to me.

File: utils/build.py (hunk 1 above)
Comment (dekimir) [inline utils/build.py]: No: see #155.

**Comment by AWoloszyn** [comment]
Right looks good to me.

**Comment by antiagainst** [comment]
Already committed as 85207c70fcb83995e5f362dbdaf4ee7b7af1e968.

**Pull request closed without merging**